pulsar-metadata/src/main/java/org/apache/pulsar/metadata/impl/RocksdbMetadataStore.java

```diff
@@ -313,6 +318,8 @@ private void configLog(Options options) throws IOException {
             logPathSetting = FileSystems.getDefault().getPath(logPath + "/rocksdb-log");
             Files.createDirectories(logPathSetting);
             options.setDbLogDir(logPathSetting.toString());
+            Set<PosixFilePermission> perms = PosixFilePermissions.fromString("rwx------");
```

Comment:
Same here.

Reply:
Hi @Jason918 , this fix and line 226 are both set permission for directory, perhaps "rwxr-x---" is better? And for file "rw-r-----" is better.

Reply:
"rwxr-x---" +1

Reply:
Thanks for review, and I found some permission problem for PULSAR-19 and PR for #15411 ,can you please take a look?